x-pack/plugins/observability_solution/apm/public/components/routing/app_root/apm_header_action_menu/add_data_context_menu.tsx

```diff
@@ -91,17 +94,23 @@ export function AddDataContextMenu() {
     },
   ];
 
+  const handleTourClose = () => {
+    setTourState({ shownBefore: true, isActive: false });
+    setPopoverOpen(false);
+  };
   return (
     <>
       <EuiPopover
         id="integrations-menu"
         button={button}
-        isOpen={popoverOpen}
+        isOpen={popoverOpen || tourState.isActive}
```

Comment:
Like shown offline, the menu is stuck open when the tour is active and the user reloads the page.